data_management.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from datetime import datetime
+
+def add_transaction():
+    while True:
+        date = input("Enter the date (YYYY-MM-DD): ")
+        try:
+            datetime.strptime(date, "%Y-%m-%d")
+        except ValueError:
+            print(f"'{datetime}' is not a valid format (YYYY-MM-DD)")
+            print("Please try again")
+            print()
+            continue
+
+        category = input("Enter the category (e.g., Food, Rent): ")
+        category = category.capitalize()
+
+        description = input("Enter a description: ")
+        description = description.capitalize()
+
+        amount = input("Enter the amount: ")
+        if amount.isnumeric():
```

Comment:
```suggestion
        if not amount.isnumeric():
            print(f"{amount} is not a valid format ")
            print("Please try again")
            print()
            continue
```

Your code works correctly, but using `if not` syntax is smarter